pages/multiselect/permutations.page.tsx

```diff
@@ -82,6 +82,16 @@ const permutations = createPermutations<MultiselectProps & { inlineTokens?: bool
     i18nStrings: [i18nStrings],
     inlineTokens: [true],
   },
+  {
+    placeholder: ['Select an item'],
+    readOnly: [true],
+    tokenLimit: [2],
+    options: [options],
+    selectedOptions: [[options[0], options[1]]],
```

Comment:
Do we need a permutation for when the number of selected tokens exceeds the `tokenLimit` as that would then show the "Show more" state?

Reply:
I don't think we need this example, because the "Show more" behavior does not change when `readOnly` is set to true

Reply:
Is that not then what the permutation would be asserting? so that in future if a change is made to the readonly styles of a button/link that it doesn't show up here. Are we also maybe missing a unit test that in a readonly state we can still click the "Show more" button?

Reply:
added u test and permutation example